src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/RealExternalServiceDAO.scala

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.rawls.model.AgoraEntity
+import org.broadinstitute.dsde.rawls.model.JobStatus
+import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport._
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import scala.util.Try
+import spray.client.pipelining._
+import spray.http._
+import spray.http.HttpHeaders.Cookie
+import spray.httpx.SprayJsonSupport._
+import spray.json._
+
+/**
+ * @author tsharpe
+ */
+class RealExternalServiceDAO( methodRepoConf: Config, execSvcConf: Config)( implicit system: ActorSystem ) extends ExternalServiceDAO {
+
+  override def getMethodConfig( namespace: String, name: String, version: String, authCookie: HttpCookie ): Try[HttpResponse] = {
```

Comment:
I disagree about the cookie.  That's the format we receive the authorization in, so all we have to do is pass it on.  Accessing it via any other format would mean unnecessary extraction and repackaging.